libraries/MySensors/MyConfig.h

```diff
@@ -101,7 +101,7 @@
 #define MY_ATSHA204_PIN 17 // A3 - pin where ATSHA204 is attached
 
 // MySigningAtsha204Soft default settings
-#define MY_RANDOMSEED_PIN 7 // A7 - Pin used for random generation (do not connect anything to this)
+#define MY_RANDOMSEED_PIN A0 // A0 - Pin used for random generation (do not connect anything to this)
```

Comment:
I don't like that the defaults are changed for a specific sketch to function. Use constructor arguments in this case.